lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -46,6 +46,13 @@ llvm::cl::opt<std::string> loadBackendSpecificOptionsOpt(
     llvm::cl::cat(hostManagerCat));
 } // namespace
 
+/// The device configs file used for Runtime.
+llvm::cl::opt<std::string> loadDeviceConfigsFileOpt(
+    "load-device-configs",
+    llvm::cl::desc("Load device configs used in Runtime"),
+    llvm::cl::value_desc("profile.yaml"), llvm::cl::Optional,
```

Comment:
`profile.yaml` is not the best value for a config file option. Maybe `config.yaml` is a better choice?